hasher-matcher-actioner/hmalib/lambdas/api/lcc.py

```diff
@@ -0,0 +1,65 @@
+from dataclasses import dataclass, asdict
+from mypy_boto3_dynamodb.service_resource import Table
+import bottle
+
+from hmalib.common.config import HMAConfig
+from hmalib.common.models.bank import BankMember, BanksTable
+
+from hmalib.lambdas.api.middleware import (
+    jsoninator,
+    JSONifiable,
+    DictParseable,
+    SubApp,
+)
+
+import typing as t
+
+
+# I return this type because I saw in the documentation that a
+# to_json method is required
+
+# I might need to use
+# `jsoninator(<requestObj>, from_query=True)` but not sure how the from_query=True
+# would modify this implementation
+@dataclass
+class LccResponse(JSONifiable):
+    found_match: bool
+    content_id: t.Optional[str]
+    preview_url: t.Optional[str]
+
+    def to_json(self) -> t.Dict:
+        return {
+            "found_match": self.found_match,
+            "content_id": self.content_id,
+            "preview_url": self.preview_url,
+        }
+
+    # def to_json(self) -> t.Dict:
+    #     return asdict(self)
+
+
+def get_lcc_api(
+    datastore_table: Table,
+    hma_config_table: str,
+    indexes_bucket_name: str,
+    writeback_queue_url: str,
+    bank_table: Table,
+    hash_queue_url: str,
```

Comment:
As mentioned elsewhere, you can remove these. We can have @franklin-phan fill em out.